### PR TITLE
Add Lab and link SampleTissue to Lab instead of Institute

### PR DESCRIPTION
**Pull request by callunity**
(work was done by @djcooke )

### CODE REVIEW

**Comment by callunity** [comment]
:+1:

File: miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js [hunk 1]
```diff
@@ -415,23 +415,20 @@ var Institute = Institute || {
       data.sort(function (a, b) { //////////////// TODO: confirm alias??
         return (a.alias > b.alias) ? 1 : ((b.alias > a.alias) ? -1 : 0);
       });
-    } // else collection is empty, so render only the "Add New" button
+    } // else collection is empty, so render only teh "Add New" button
```
Comment (apmasell) [inline miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js]: I don't really care, but you introduce a typo?

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js (hunk 1 above)
Comment (callunity) [inline miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js]: Must've been my fault, not @djcooke . I'll be reworking Institute and Lab anyway, so let's merge this commit and I promise to fix the typo in another PR

**Pull request merged**